.github/workflows/ci-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+name: Node.js CI
+
+on: [push]
```

Comment:
Looks like you need to add `pull_request` here: `on: [push, pull_request]` as per https://futurestud.io/tutorials/github-actions-run-on-pull-request#:~:text=Run%20Actions%20on%20Pull%20Requests,-When%20creating%20a&text=Search%20the%20line%20on%3A%20%5Bpush,%5D%20and%20you're%20done.

Reply:
:( no, this did not help either. Are you sure there is no option where you can accept ci-workflow runs from new contributors?